dashdotdb/services/deploymentvalidation.py

```diff
@@ -34,15 +36,16 @@ def _insert(self, item):
             self.log.error('skipping validation: key "value" not found')
             return
 
-        expire = datetime.now() - timedelta(minutes=50)
-        db_token = db.session.query(Token) \
-            .filter(Token.timestamp > expire).first()
+        db_token = db.session.query(Tokens) \
+            .filter(Tokens.uuid == token).first()
```

Comment:
you will also need to validate the token being properly scoped

Reply:
Yes, good catch.  Fixed.